routes/tests/shopifyAuth.test.js

```diff
@@ -0,0 +1,58 @@
+const request = require('supertest');
+const http = require('http');
+const express = require('express');
+
+const {MemoryStrategy} = require('../../strategies');
+const createShopifyAuthRouter = require('../shopifyAuth');
+
+let server;
+describe('shopifyAuth', async () => {
+  afterEach(() => {
+    server.close();
+  });
+
+  describe('/', () => {
+    it('responds to get requests by returning a redirect page', async () => {
+      const {app, server} = await createServer();
+      const {status, text} = await request(app).get('/?shop=shop1').then((resp) => resp);
```

Comment:
Would it be clearer to pass `() => {}` to `then`?

Reply:
Have to return the wrapped value or else the await won't give us what we want. :/